### PR TITLE
Release to main

### PR DESCRIPTION
**Pull request by AntonReinhard**
I'm not sure if a patch version increase is enough, but technically no breaking changes were introduced. The QuantumElectrodynamics.jl integration tests might fail though.

Do not squash.

### CODE REVIEW

**Review by szabo137 — DISMISSED** [review]
LGFM. 

Why do you think, the integration tests with `QuantumElectrodynamics.jl` will fail? Still a bug in the CI?

**Comment by AntonReinhard** [comment]
> LGFM.
> 
> Why do you think, the integration tests with `QuantumElectrodynamics.jl` will fail? Still a bug in the CI?

No, the CI works fine. But as you can see in the release integration test of QED.jl, the test fails because fields has not been released again yet, so it doesn't work. https://gitlab.com/hzdr/qedjl-project/QEDevents-jl/-/jobs/8219785901

I think we should just increase the minor version after all.

**Review by szabo137 — APPROVED** [review]

File: Project.toml [hunk 1]
```diff
@@ -1,26 +1,35 @@
...
-QEDbase = "0.2.2"
-QEDcore = "0.1"
-julia = "1.6"
+QEDbase = "0.3"
+QEDcore = "0.2"
+StatsBase = "0.34"
+julia = "1.10"
 
 [extras]
+LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
```
Comment (AntonReinhard) [inline Project.toml:29]: Actually, does this need a compat entry to be registered? Do you know @szabo137 ?

Comment (AntonReinhard) [reply Project.toml:29]: Okay I think it should be fine since it's a standard library, so it doesn't seem to require a compat entry

**Pull request merged**